spyder/plugins/ipythonconsole/utils/websocket_client.py

```diff
@@ -735,13 +756,18 @@ async def _receiver_loop(self):
                     )
 
                 await self._queues[channel].put(msg)
+        except asyncio.CancelledError:
+            _LOGGER.debug("Receiver loop cancelled for %s", self.session.session)
```

Comment:
```suggestion
            _LOGGER.debug(
                "Receiver loop cancelled for %s", self.session.session
            )
```

Make this line fit in 79 columns.